dnsconfig.js

```diff
@@ -19,6 +19,96 @@ function loadTemplate(template) {
   };
 }
 
+/**
+ * Create A, optional AAAA, and optional HTTPS (SVCB) records for an OSM web service,
+ * with optional Cloudflare proxy support.
+ *
+ * @param {string} name                - Hostname (e.g. "www").
+ * @param {string[]} ipv4s             - Required array of IPv4 addresses.
+ * @param {string[]} [ipv6s=[]]        - Optional array of IPv6 addresses (defaults to empty).
+ * @param {boolean} [enableHttps=true] - Whether to create an HTTPS record (default: true).
+ * @param {boolean} [cloudflareProxy=false] - Whether to enable Cloudflare proxy (default: false).
+ *
+ */
+function osm_web_service(
+  name,
+  ipv4s,
+  ipv6s,
+  enableHttps,
+  cloudflareProxy
```

Comment:
Rather than a list of boolean arguments, which requires you to pad defaults when you need to provide a later one, and which leads to meaning lists of true and false at the call site I'd suggest taking an options hash here so the caller can pass as little or as much as it likes.

Reply:
Done.